src/runtime/refiner.ts

```diff
@@ -343,6 +348,28 @@ export class BinaryExpression extends RefinementExpression {
     }
   }
 
+  normaliseOperators(): RefinementExpression {
```

Comment:
Can you open a PR to change from AU+UK spelling to US for normali[sz]e?

Reply:
Ah yes, I'll do that!